plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -575,20 +611,18 @@ class NewCreateServiceModal extends Component {
         {
           className: 'button-primary flush-vertical',
           clickHandler: this.handleServiceReview,
-          disabled: errors.length !== 0,
+          disabled: false,
```

Comment:
Michael found this out and is fixed on the next PR in chain: https://github.com/dcos/dcos-ui/pull/1839#discussion_r98755089